package.json

```diff
@@ -11,6 +11,7 @@
     "semantic-release": "semantic-release"
   },
   "devDependencies": {
+    "@continuous-auth/semantic-release-npm": "^1.0.3",
```

Comment:
I'm unfamiliar with this dep. @DeerMichel do you have experience with this release setup and those plugins?

I usually use `npx semantic-release-cli setup` to configure this stuff, but if you're confident this setup will work for doing releases I'm okay with it.

Reply:
I guess @MarshallOfSound has 😄. All i did was following the [CFA guide](https://github.com/electron/cfa#set-up).